build.gradle.kts

```diff
@@ -24,7 +24,7 @@ object Versions {
     const val nimbusjosejwt = "9.18"
     const val postgresEmbedded = "0.13.4"
     const val postgres = "42.3.2"
-    const val redisEmbedded = "0.8.2"
+    const val redisEmbedded = "0.7.3"
```

Comment:
Måtte dessverre rulle tilbake denne oppgraderingen også for å få testene til å kjøre lokalt.

Reply:
Ser det er noe windows-krøll der og. Jeg tok og å revertet dette på isnarmesteleder,  syfoperson og syfobehandlendeenhet nå.